macros/sfdc_formula_view.sql

```diff
@@ -7,13 +7,17 @@
     )
 }}
 
+-- Raise a compiler error if users are trying to use full_statement_version=false. We are keeping the variable in the macro, however, since we don't want errors if they previously set it to true.
+{% if not full_statement_version %}
+{{ exceptions.raise_compiler_error("\ERROR: Parameter full_statement_version=false is no longer supported. Update your " ~ this.identifier|upper ~ " model to remove the full_statement_version parameter.\n") }}
```

Comment:
Suggestion to reword the warning slightly.

```suggestion
{{ exceptions.raise_compiler_error("\nERROR: The 'full_statement_version=false' parameter is no longer supported. Please update your " ~ this.identifier|upper ~ " model to remove this parameter.\n") }}
```

Reply:
Updated!